src/compiler/builder.ts

```diff
@@ -311,7 +313,7 @@ namespace ts {
         newState.affectedFilesIndex = state.affectedFilesIndex;
         newState.currentChangedFilePath = state.currentChangedFilePath;
         newState.currentAffectedFilesSignatures = state.currentAffectedFilesSignatures && new Map(state.currentAffectedFilesSignatures);
-        newState.currentAffectedFilesExportedModulesMap = state.currentAffectedFilesExportedModulesMap && new Map(state.currentAffectedFilesExportedModulesMap);

```

Comment:
@andrewbranch I retained the existing behavior but, now that I look at it, it seems wrong.  The value type of this map is a set - doesn't that mean that the new and old maps will refer to (and modify) the same sets?  It seems like the intention was probably to do a deep copy.